src/main/java/net/pms/util/ProcessUtil.java

```diff
@@ -217,8 +217,13 @@ public static String run(String... cmd) {
 	// Whitewash any arguments not suitable to display in dbg messages
 	// and make one single printable string
 	public static String dbgWashCmds(String[] cmd) {
-		for(int i=0; i < cmd.length; i++) {
-			if(cmd[i].contains("headers")) {
+		for (int i=0; i < cmd.length; i++) {
+			// Wrap arguments with spaces in double quotes to make them runnable if copy-pasted
+			if (cmd[i].contains(" ")) {
```

Comment:
@SubJunk This function is only for debug printing. It can't break anything as far as I can tell. The whole purpose was to make it so that one can copy-paste the command found in the log and it will actually work as intended. Processes run from Java send arguments as an array, not a space-separated string, but we turn it into a space separated string when logging it. I just made sure that if any of the arguments, one of the elements in the array, contains space, then that element is wrapped in double quotes before added to the debug string for logging. Has this created any problems?